UPGRADE-3.1.md

```diff
@@ -88,6 +88,27 @@ FrameworkBundle
    cache service. If you are using `serializer.mapping.cache.apc`, use
    `serializer.mapping.cache.doctrine.apc` instead.
 
+ * The `framework.serializer.cache` option has been deprecated. Configure a cache pool
+   called `serializer` under `framework.cache.pools` instead.
+
+   Before:
+
+   ```yaml
+   framework:
+       serializer:
+           cache: serializer.mapping.cache.apc
+   ```
+
+   After:
+
+   ```yaml
+   framework:
+       serializer: ~
```

Comment:
no it should not be removed. If you remove the ``serializer`` key entirely, it disables the serializer in FrameworkBundle